ipm_interfaces/package.xml

```diff
@@ -12,11 +12,11 @@
 
   <exec_depend>rosidl_default_runtime</exec_depend>
 
+  <depend>builtin_interfaces</depend>
   <depend>geometry_msgs</depend>
   <depend>sensor_msgs</depend>
   <depend>shape_msgs</depend>
   <depend>vision_msgs</depend>
-  <depend>std_msgs</depend>
```

Comment:
The standard msgs are still used in the code.

Reply:
std_msgs isn't used in ipm_interfaces anymore. It is used in ipm_library, which still has std_msgs in the package.xml

Reply:
Oh right